lib/internal/modules/esm/hooks.js

```diff
@@ -761,6 +701,11 @@ function pluckHooks({
 
   if (initialize) {
     acceptedHooks.initialize = initialize;
+  } else if (globalPreload) { // TODO(JakobJingleheimer): Remove this when loaders go "stable".
+    process.emitWarning(
+      '`globalPreload` has been removed; use `initialize` instead.',
+      'DeprecationWarning',
```

Comment:
I don't think we do deprecation warnings for experimental features. I think this would just be a thrown error.

Reply:
It shouldn't be a thrown error, a warning is the right choice, although not a `DeprecationWarning`

Reply:
What kind of warning should it be?

Reply:
I see other warnings in the codebase using `UnsupportedWarning` as a type, which seems appropriate.